paramak/parametric_shapes/sweep_circle_shape.py

```diff
@@ -14,8 +14,12 @@ class SweepCircleShape(Shape):
         path_points (list of tuples each containing X (float), Z (float)): A list of XY,
             YZ or XZ coordinates connected by spline connections which define the path
             along which the 2D shape is swept.
-        workplane (str): Workplane in which the circle to be swept is defined.
-        path_workplane (str): Workplane in which the spline path is defined.
+        workplane (str): Workplane in which the circle to be swept is defined. Defaults
+            to "XY".
```

Comment:
I don't know how you guys see it but in #357 I didn't rewrite the descriptions that live in the inherited class. Like for stp_filename and stl_filename below. Maybe that's not the way to go though
```suggestion
        workplane (str, optional): Defaults to "XY".
```